redis/src/main/scala/zio/redis/options/Scripting.scala

```diff
@@ -0,0 +1,6 @@
+package zio.redis.options
+
+trait Scripting {
+
+  case class Script[K, A](lua: String, keys: Seq[K], args: Seq[A])
```

Comment:
Let's not use `Chunk` instead of `Seq`. Also, make it sealed. More important, this isn't really an option.

Reply:
Sorry, i don't understand that comment((

Reply:
Sorry, my english broke, I wanted to say let's use `Chunk` instead of `Seq` and let's avoid using `Seq`.